src/utils/setup.ts

```diff
@@ -0,0 +1,266 @@
+/* eslint @typescript-eslint/no-var-requires: "off" */
+import { ExchangeRate, Bitcoin, BTCUnit, Polkadot, PolkadotUnit, BTCAmount } from "@interlay/monetary-js";
+import { Big } from "big.js";
+import BN from "bn.js";
+import { createPolkadotAPI } from "../factory";
+import { issueSingle, REGTEST_ESPLORA_BASE_PATH, setNumericStorage } from ".";
+import { ApiPromise, Keyring } from "@polkadot/api";
+import {
+    DefaultNominationAPI,
+    DefaultOracleAPI,
+    DefaultRedeemAPI,
+    NominationAPI,
+    OracleAPI,
+    RedeemAPI,
+    TransactionAPI,
+} from "../parachain";
+import { BitcoinCoreClient } from "./bitcoin-core-client";
+import { DefaultElectrsAPI, ElectrsAPI } from "../external";
+import { KeyringPair } from "@polkadot/keyring/types";
+import * as bitcoinjs from "bitcoinjs-lib";
+import { cryptoWaitReady } from "@polkadot/util-crypto";
+
+export const DEFAULT_PARACHAIN_ENDPOINT = "ws://127.0.0.1:9944";
+export const DEFAULT_FAUCET_ENDPOINT = "http://0.0.0.0:3036";
+export const DEFAULT_BITCOIN_CORE_NETWORK = "regtest";
+export const DEFAULT_BITCOIN_CORE_HOST = "0.0.0.0";
+export const DEFAULT_BITCOIN_CORE_USERNAME = "rpcuser";
+export const DEFAULT_BITCOIN_CORE_PASSWORD = "rpcpassword";
+export const DEFAULT_BITCOIN_CORE_PORT = "18443";
+export const DEFAULT_BITCOIN_CORE_WALLET = "Alice";
+export const DEFAULT_REDEEM_ADDRESS = "bcrt1qed0qljupsmqhxul67r7358s60reqa2qtte0kay";
+
+// Command line arguments of the initialization script
+const yargs = require("yargs/yargs");
+const { hideBin } = require("yargs/helpers");
+const argv = yargs(hideBin(process.argv))
+    .option("initialize", {
+        type: "boolean",
+        description: "Flag that decides whether this script should be run or not",
+        default: false,
+    })
+    .option("set-stable-confirmations", {
+        type: "boolean",
+        description: "Set the required bitcoin and parachain confirmations to zero",
```

Comment:
Description here is misleading, if I'm correct the caller can set any parameter they want?

Reply:
I actually think this is not worth including if we can query from the parachain - i.e. on dev it will be zero and there is no need to set this param otherwise.

Reply:
> Description here is misleading
The CLI can't set number of confirmations, only whether to set the to zero

> on dev it will be zero and there is no need to set this param otherwise.
It's not zero on local in my experience, the tests fail if this isn't run

Reply:
> It's not zero on local in my experience, the tests fail if this isn't run

It's probably one, but this shouldn't make a difference. If the tests fail because of this we should find out why.

Reply:
It's 1 bitcoin confirmation but 100 parachain confirmations

Reply:
Ah that's right, perhaps we should anyway set both to zero for dev in genesis.